source/includes/inkling2-reference/_algorithms.html.md

```diff
@@ -71,5 +71,8 @@ In addition to the first (shared) table of parameters, PPO supports these additi
 
 | Parameter (Example use)      | Description |
 | -                            | -           |
+| BatchSize: (1000-2000000) <br> (`BatchSize: 8000`) | See below for details. |
 | PolicyLearningRate: float <br> (`PolicyLearningRate: 0.0001`) | The learning rate for training the policy network. |
 
+BatchSize Details:
+Proximal Policy Optimization works by gathering a large number of complete trajectories and analyzing them in aggregate to obtain a confidence metric for the probability that a given change to the policy will improve performance. With a sufficiently large batch size, this yields monotonic policy improvement, as the algorithm only makes changes it’s confident will lead to real improvements based on large amounts of data. The BatchSize parameter determines how much data will be aggregated to make this decision. Smaller batches will lead to faster convergence as updates will be computed more frequently, but these updates will be less reliable and the policy may become unstable if this value is too small. We recommend you set this value to be large enough to contain at least one full episode from each of your simulators, as full-episode trajectories are the most valuable for estimating policy performance. For example, if your episodes are up to 100 steps each and you have 100 simulators, try a value of 10000.
```

Comment:
```suggestion
Proximal Policy Optimization works by gathering a large number of complete trajectories and analyzing them in aggregate to obtain a confidence metric for the probability that a given change to the policy will improve performance. With a sufficiently large batch size, this yields monotonic policy improvement, as the algorithm only makes changes it’s confident will lead to real improvements based on large amounts of data. The BatchSize parameter determines how much data will be aggregated to make this decision. Smaller batches will lead to faster convergence as updates will be computed more frequently, but these updates will be less reliable and the policy may become unstable if this value is too small. We recommend you set this value to be large enough to contain a significant fraction of a complete episode from each of your simulators, as that aids in estimating policy performance. For example, if your episodes are up to 100 steps each and you have 100 simulators, try a value between 5000-10000.
```

Reply:
@Random-Word thoughts on proposed rewording ^^&?

Reply:
That's good. Mostly I'm not sure how they should scale the batch size with number of sims. Maybe it is as simple as 10k works for 100 sims, 20k works for 200 sims, etc.